scripts/signature.py

```diff
@@ -210,6 +214,46 @@ def bias_signatures(base_signatures, contender_signatures, tickets, error_path):
     return "", status
 
 
+def bias_aggregates(args):
+    """
+    Finds and exports aggregates whose signatures have been modified agasint a baseline.
+    Saves the results to a file and sets a Github Actions Output.
+    Currently this is hardcoded to presto aggregates.
+    """
+    with open(args.base) as f:
+        base_signatures = json.load(f)
+
+    with open(args.contender) as f:
+        contender_signatures = json.load(f)
+
+    delta, status = diff_signatures(
+        base_signatures, contender_signatures, args.error_path
+    )
+
+    set_gh_output(f"presto_aggregate_error", status == 1)
+
+    if not delta:
+        print(f"{bcolors.BOLD} No changes detected: Nothing to do!")
+        return "", status
+
+    function_set = set()
+    for items in delta.values():
+        for item in items:
+            fn_name = item.get_root_key()
+            pattern = aggregate_pattern.match(fn_name)
+            if pattern:
+                function_set.add(pattern.group(1))
+            else:
+                function_set.add(fn_name)
+
+    if function_set:
+        biased_functions = ",".join(function_set)
+        with open(args.output_path, "w") as f:
+            print(f"{biased_functions}", file=f, end="")
+
+        set_gh_output(f"presto_aggregate_functions", True)
```

Comment:
```suggestion
        set_gh_output("presto_aggregate_functions", True)
```
Same here, no need to prefix with `f` if your not doing any formatting :)